api/location/schema.py

```diff
@@ -72,8 +74,9 @@ def mutate(self, info, location_id, **kwargs):
             validate_url(**kwargs)
         if "abbreviation" in kwargs:  # noqa
             validate_empty_fields(**kwargs)
-        result = location.filter(LocationModel.name == kwargs.get('name'))
-        if result.count() > 0:
+        active_locations = result.filter(
+            LocationModel.name == kwargs.get('name'))
+        if active_locations.count() > 0:
             pass
```

Comment:
@michael-basweti you can just do `if active_locations.count()` instead of `if active_locations.count() > 0`

Reply:
@kampkelly We need this for readability and also we should have an active location for operation to go through